heartbeat/apache

```diff
@@ -242,9 +243,17 @@ apache_start() {
 	validate_default_config || return $OCF_ERR_CONFIGURED
 
 	if [ -z $PIDFILE_DIRECTIVE ]; then
-		ocf_run $HTTPD $HTTPDOPTS $OPTIONS -f $CONFIGFILE
+		if [ -z "$LOAD_STATUS_MODULE" ]; then
```

Comment:
This seems to do what `HTTPDOPTS="$HTTPDOPTS -C $LOAD_STATUS_MODULE"` already did, but in 2 places which makes it more prone to future errors.

Reply:
It does not do it. If I do it so you have proposed bash makes a lot of sensless quoting and httpd does not start. Do not forget $LOAD_STATUS_MODULE already contains quotes. I've already spent a lot of time to find the right solution but this seemed to me to be the most lear and stable.

Reply:
Have you tried changing it to `HTTPDOPTS="$HTTPDOPTS -C \"$LOAD_STATUS_MODULE\""`?

Reply:
Yes I've. Result is:
... -C '"LoadModule' status_module '/usr/lib64/apache2-prefork/mod_status.so"'  ...
instead of:
... -C 'LoadModule status_module /usr/lib64/apache2-prefork/mod_status.so"  ...
And apache2 can not start. I've tested it with the combination of single quote too. Noting hase helped.